src/Strategy/Erlang/ConfigParser.hs

```diff
@@ -0,0 +1,51 @@
+module Strategy.Erlang.ConfigParser (
+  parseConfig,
+  parseErlValue,
+) where
+
+import Data.Functor (void)
+import Data.Text (Text)
+import Data.Void
+import Text.Megaparsec
+import Text.Megaparsec.Char
+import qualified Text.Megaparsec.Char.Lexer as L
+
+type Parser = Parsec Void Text
+
+data ErlValue
+  = Atom Text
+  | Array [ErlValue]
+  | Pair Text ErlValue
+
+parseConfig :: Parser [ErlValue]
+parseConfig = do
+  scn
+  many (parsePair <* char '.')
```

Comment:
this probably isn't correct -- see https://stackoverflow.com/questions/1110601/in-erlang-when-do-i-use-or-or for what `.` and `,` mean in erlang. also: `sepBy` might help here

Reply:
This does seem correct to me.  Every rebar example I can find terminates with a `.`.  Also, the link you posted is erlang syntax, rather than erlang config syntax, which seems to be a slightly altered subset of erlang syntax.  Functions, for example, are not allowed in erlang config.

Reply:
oh you're right here -- I misunderstood what was happening. This is definitely real erlang syntax though, even if some things aren't allowed